packages/react/src/components/form/NativeSelect/NativeSelect.module.css

```
@@ -0,0 +1,64 @@
+.input {
+  font: inherit;
+  position: relative;
+  box-sizing: border-box;
+  flex: 0 1 auto;
+  min-height: 2.5em;
+  width: 100%;
+  padding: 0 var(--fds-spacing-3);
```

Looked like to much padding with `spacing-3`, suggest `spacing-2`
```suggestion
  padding: 0 var(--fds-spacing-2);
```